x-pack/solutions/observability/plugins/observability_onboarding/public/application/quickstart_flows/auto_detect/auto_detect_panel.tsx

```diff
@@ -183,6 +183,12 @@ export const AutoDetectPanel: FunctionComponent = () => {
                                       href: assetDetailsLocator.getRedirectUrl({
                                         assetType: 'host',
                                         assetId: integration.metadata?.hostname,
+                                        assetDetails: {
```

Comment:
What is the current default time range?

Reply:
Also from -15 minutes to now, but it's not relative, once user navigates to the page it get converted to an absolute timestamp and never change